csharp/tools/tracing-config.lua

```diff
@@ -0,0 +1,55 @@
+function RegisterExtractorPack()
+    local extractor = GetPlatformToolsDirectory() ..
+                          'Semmle.Extraction.CSharp.Driver'
+    if OperatingSystem == 'windows' then
+        extractor = GetPlatformToolsDirectory() ..
+                        'Semmle.Extraction.CSharp.Driver.exe'
```

Comment:
I don't know lua, but could you simply append `.exe` to `extractor` in this case?

Reply:
Good point, yes that's possible